Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN cd /tmp && \
     if [[ "$(cat .git/HEAD)" == "ref:"* ]]; then sha=$(cat .git/$sha); fi && \
     echo "$(date +"%Y%m%d%H%M%S")-$sha" > /tmp/BUILD
 
-FROM centos/postgresql-10-centos8:latest
+FROM postgresql-container:10
```

Comment:
I think there's a nicer way to represent a "local" image, but I'm not sure what that would be.